src/pybind/mgr/cephadm/services/cephadmservice.py

```diff
@@ -73,6 +75,61 @@ def simplified_keyring(entity: str, contents: str) -> str:
     return keyring
 
 
+def get_dashboard_eps(svc: 'CephadmService') -> Tuple[List[str], Optional[str]]:
```

Comment:
Not a big deal, but in general in Python you should favor long/descriptive variable names over nicknamed ones. `mgmt` is a well-known abbreviation, but `eps` is not.

Additionally, for consistency: in some places I see `get_service_endpoints` vs. `get_dashboard_eps`.